CHANGELOG.next.md

```diff
@@ -9,6 +9,11 @@ Thanks, you're awesome :-) -->
 ## Unreleased
 
 ### Schema Changes
+
+#### Added
```

Comment:
I hadn't noticed this section header was missing, and I've been putting my schema changes under `Tooling`! 😅

Thanks for catching it! We'll make sure the other sections are cleaned up.